### PR TITLE
Add model for clinic and test.

### PR DESCRIPTION
**Pull request by patonit**
Please review my code.

### CODE REVIEW

File: src/main/java/pl/paton/rafal/java/app/ar/model/Clinic.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
+package pl.paton.rafal.java.app.ar.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Clinic {
+
+//    private String street;
```
Comment (juniorjavadeveloper-pl) [inline src/main/java/pl/paton/rafal/java/app/ar/model/Clinic.java:8]: Please remove comments!

File: src/main/java/pl/paton/rafal/java/app/ar/model/Clinic.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    private String name;
+    private Address address;
+    private List<Client> clients = new ArrayList<>();
+    // doctors
+    // appointments
+    // patients
+    // clients
+
+    public boolean registerClient(Client client) {
+        //System.out.println("Registering client");
```
Comment (juniorjavadeveloper-pl) [inline src/main/java/pl/paton/rafal/java/app/ar/model/Clinic.java:23]: Avoid sout!

File: src/test/java/pl/paton/rafal/java/app/ar/model/ClinicTest.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        // when - wywołanie metody testowanej oraz przypisanie wartości zwracanej z metody
+        boolean registered = clinic.registerClient(null);
+
+        // then - weryfikacja poprawności zwracanej wartości metody testowanej
+        Assertions.assertFalse(registered, "Client registered in clinic");
+
+    }
+
+    @Test
+    void register() {
```
Comment (juniorjavadeveloper-pl) [inline src/test/java/pl/paton/rafal/java/app/ar/model/ClinicTest.java:30]: Please finish test!

**Review by juniorjavadeveloper-pl — CHANGES_REQUESTED** [review]
Please make changes!

File: src/test/java/pl/paton/rafal/java/app/ar/model/ClinicTest.java (hunk 3 above)
Comment (patonit) [reply src/test/java/pl/paton/rafal/java/app/ar/model/ClinicTest.java:30]: Done.

File: src/main/java/pl/paton/rafal/java/app/ar/model/Clinic.java (hunk 1 above)
Comment (patonit) [reply src/main/java/pl/paton/rafal/java/app/ar/model/Clinic.java:8]: No way I need them.

File: src/main/java/pl/paton/rafal/java/app/ar/model/Clinic.java (hunk 2 above)
Comment (patonit) [reply src/main/java/pl/paton/rafal/java/app/ar/model/Clinic.java]: Done.

**Review by juniorjavadeveloper-pl — APPROVED** [review]
Good job!

**Pull request merged**